source/_integrations/zwave_js.markdown

```diff
@@ -148,76 +149,77 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
 2. Select the **Z-Wave** integration.
    - Then, select the cogwheel {% icon "mdi:cog-outline" %}.
 3. Select **Remove a device**, then **Start exclusion**.
-   - The Z-Wave controller is now in exclusion mode.
+   - The Z-Wave adapter is now in exclusion mode.
 4. Put the device you want to remove in exclusion mode. Refer to its manual how this is done.
 5. The UI should confirm that the device was removed and the device and entities will be removed from Home Assistant.
 
-## Migrating a Z-Wave network to a new controller
+## Migrating a Z-Wave network to a new adapter
 
-Do this if you have an existing Z-Wave network and want to use a new controller. This will reset your current controller (remove all network information from it) and remove the controller from Home Assistant. The Z-Wave integration with all its entities will stay in Home Assistant. The new controller is added to Home Assistant and paired with the existing network.
+Do this if you have an existing Z-Wave network and want to use a new adapter. This will reset your current adapter (remove all network information from it) and remove the adapter from Home Assistant. The Z-Wave integration with all its entities will stay in Home Assistant. The new adapter is added to Home Assistant and paired with the existing network.
 
 ### Prerequisites
 
 - Administrator rights in Home Assistant
-- If you want to migrate from a 500 series controller, before starting migration, update the controller to SDK 6.61+
+- If you want to migrate from a 500 series adapter, before starting migration, update the adapter to SDK 6.61+
   - Check the documentation of your device to see if and how they can be updated.
   - [Steps to update Aeotec Z-Stick 5](https://aeotec.freshdesk.com/support/solutions/articles/6000252294-z-stick-gen5-v1-02-firmware-update).
 
-### To migrate a Z-Wave network to a new controller
+### To migrate a Z-Wave network to a new adapter
 
 1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & services**" %}.
 2. Select the **Z-Wave** integration.
    - Then, select the cogwheel {% icon "mdi:cog-outline" %}.
-3. Under **Backup and restore**, select **Migrate controller**.
-4. Select **Migrate to a new controller**.
+3. Under **Backup and restore**, select **Migrate adapter**.
+4. Select **Migrate to a new adapter**.
+
    - To confirm device reset, select **Submit**.
-   - **Info**: This will initiate a backup of the network information and factory reset the controller. All the stored network information will be removed.
-5. When the **Unplug your controller** dialog shows up, unplug your old controller.
-   - Connect the new controller.
-   - Confirm that you connected the new controller by selecting **Submit**.
+   - **Info**: This will initiate a backup of the network information and factory reset the adapter. All the stored network information will be removed.
```

Comment:
Side note: We now no longer reset the old adapter during the migration. We didn't update the docs yet to reflect that.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/39823